compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -180,41 +180,8 @@ generateSrcFromLf env thisPkgId = noLoc mod
                 | conName <- cons
                 ]
 
-    imports = declImports ++ additionalImports
-    mkImport :: Bool -> String -> [LImportDecl GhcPs]
-    mkImport pred modName = [ noLoc $
-        ImportDecl
-            { ideclExt = noExt
-            , ideclSourceSrc = NoSourceText
-            , ideclName = noLoc $ mkModuleName modName
-            , ideclPkgQual = Nothing
-            , ideclSource = False
-            , ideclSafe = False
-            , ideclImplicit = False
-            , ideclQualified = False
-            , ideclAs = Nothing
-            , ideclHiding = Nothing
-            } :: LImportDecl GhcPs
-        | pred
-        ]
-    -- additional imports needed for typechecking
-    additionalImports =
```

Comment:
(just curious) why these `additionalImports` are no longer needed?

Reply:
we used to lift daml-prim and daml-stdlib data types, now we just always point it at the compiling sdk libraries.